app/models/pipeline_run.rb

```diff
@@ -1580,14 +1580,11 @@ def outputs_by_step(can_see_stage1_results = false)
           end
           file_info << file_info_for_output
         end
-        if file_info.present?
-          result[prs.name]["steps"][target_name] = {
-            "step_description" => STEP_DESCRIPTIONS[prs.name]["steps"][target_name],
-            # Do not send list of files.
-            # "file_list" => file_info,
-            "reads_after" => (job_stats_by_task[target_name] || {})["reads_after"],
-          }
-        end
+
```

Comment:
This is to enable the "reads remaining" table. We don't need the `file_info` check in this case, so it's safer to leave off the unneeded files.